plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-checkout-block-calculate-tax.spec.js

```diff
@@ -111,6 +112,7 @@ test.describe(
 		// eslint-disable-next-line playwright/expect-expect
 		test( 'can create Cart Block page', async ( { page } ) => {
 			await goToPageEditor( { page } );
+			await closeChoosePatternModal( { page } );
```

Comment:
This is basically being used after each `goToPageEditor`, we might as well include it in  it.

Reply:
I thought about the same

Reply:
Done in a new PR https://github.com/woocommerce/woocommerce/pull/50464